tracer/src/Datadog.Trace.Tools.dd_dotnet/CommandWithExamples.cs

```diff
@@ -19,6 +20,10 @@ public CommandWithExamples(string name, string? description = null)
     {
     }
 
+    public static string Command =>
+        Environment.GetEnvironmentVariable("DD_OVERRIDE_COMMAND")
```

Comment:
Given this is and internal only variable, I wonder if we should use `DD_INTERNAL`/`_DD` as we have for other internal variables. Mostly to avoid any risk of future conflict, and to make it clear it's not meant for customers to play with?

Reply:
Good point